tools/manta.cwl

```diff
@@ -0,0 +1,96 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: manta 
+doc: "Calls structural variants."
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: $(inputs.ram * 1000)
+    coresMin: $(inputs.cpu)
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/manta:1.6.0'
+  - class: InitialWorkDirRequirement
+    listing:
+    - entryname: rename_outs.sh
```

Comment:
Out of curiosity, why choose to make the script here instead of import a standalone script?

Reply:
Because this is a very specific script that's only useful to manta. Felt appropriate to have it here.